packml_sm/include/packml_sm/abstract_state_machine.h

```diff
@@ -321,13 +321,13 @@ class AbstractStateMachine
    * @brief Call to increment or add a specific Itemized error stat.
    *
    */
-  void incrementErrorStatItem(int16_t id, int32_t count, double duration);
+  void incrementErrorStatItem(int16_t id, double count, double duration);
```

Comment:
Question: should ROS msg float32 be mapped to float or double (or do we just always default to double)?

Reply:
Ah! Excellent point. Fixed.